scheduler/views.py

```diff
@@ -22,13 +22,18 @@ class CreatedAtCursorPagination(CursorPagination):
     ordering = "-created_at"
 
 
+class IdCursorPagination(CursorPagination):
+    ordering = "-id"
```

Comment:
@rudigiesler that seems fine for users and groups but wouldn't we want the failed-tasks to display the most recent ones first?